Mathlib/Algebra/Polynomial/Basic.lean

```diff
@@ -1190,7 +1196,7 @@ instance ring : Ring R[X] :=
     toIntCast := Polynomial.intCast
     toNeg := Polynomial.neg'
     toSub := Polynomial.sub
-    zsmul := ((· • ·) : ℤ → R[X] → R[X]) }
+    zsmul := ((. • .) : ℤ → R[X] → R[X]) }
```

Comment:
Are you partially reverting #12146 here and elsewhere?

Reply:
Yes, this file has lots of bad changes, I'm going to revert it. Sorry.